console/console-init/ui/src/modules/msg-and-iot/components/IoTReview/IoTReview.tsx

```diff
@@ -118,7 +118,7 @@ export const IoTReview: React.FunctionComponent<IIoTReviewProps> = ({
                   <Button
                     id="preview-iot-copy-button"
                     variant={ButtonVariant.link}
-                    aria-label="copy-iot-config"
+                    aria-label="copy iot configuration"
                     onClick={() => {
```

Comment:
don't need extra curly braces, remove extra braces.
onClick={() =>setIsCopied(true)}

Reply:
It will be modified in future. Once we start integrating it with mockserver